src/features/filters/ReviewFilter.jsx

```diff
@@ -9,29 +9,38 @@ import { CheckboxWrapper } from '../../components/CheckboxWrapper';
 
 const ReviewFilter = () => {
   const dispatch = useDispatch();
-  const reviewed = useSelector(selectReviewed);
-  const notReviewed = useSelector(selectNotReviewed);
-  const includeReviewed = reviewed === null || reviewed;
-  const includeNotReviewed = notReviewed === null || notReviewed;
+  let reviewed = useSelector(selectReviewed);
 
+  /*
+    Reviewed can have 3 states: null, true, false. 
+      1. Null: both reviewed and not reviewed images are included.
+      2. True: Only reviewed images are included.
+      3. False: only not-reviewed images are included.
+    Using these 3 states we can then manage both the filter state and the UI state.
+  */
```

Comment:
This new state we are managing also prevents both filters from being false(no images would fall under this category). At first glance, it may see over complicated, but I think makes the state more intuitive.